src/pages/ContentScripts/DeveloperNetwork.tsx

```diff
@@ -137,7 +137,7 @@ const DeveloperNetworkView: React.FC<DeveloperNetworkViewProps> = ({
   };
 
   const graphStyle = {
-    width: '400px',
+    width: '560px',
```

Comment:
emm, @debjit20504, I'm afraid that the width is still fixed, which is just changed from a fixed '400px' to another fixed '560px'.

if you adjust your browser's width then refresh the page, you will find the problem still exists.

Reply:
> emm, @debjit20504, I'm afraid that the width is still fixed, which is just changed from a fixed '400px' to another fixed '560px'.
> 
> if you adjust your browser's width then refresh the page, you will find the problem still exists.

Ok, let me check

Reply:
@tyn1998  can you help me with this 
```typescript
  const heightOutput = document.querySelector('#height');
  const widthOutput = document.querySelector('#width');

  function reportWindowSize() {
    heightOutput.textContent = window.innerHeight;
    widthOutput.textContent = window.innerWidth;
  }
```
in this it's throwing error on heightOutput.textContent  and widthOutput.textContent 
```
Type 'number' is not assignable to type 'string'.
```

Reply:
It is a simple type unmatch error since we use typescript to develope Hypercrx.

In fact the issue can be solved very easily, I'll give you some hint in another comment.